docs/cep/query-guide/functions/streaming-ml/bayesianregression.md

```diff
@@ -30,16 +30,47 @@ streamingml:bayesianRegression(<STRING> model.name, <INT> prediction.samples, <D
 
 ## Example 1
 
-    CREATE STREAM StreamA (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double);
-
-    from StreamA#streamingml:bayesianRegression('model1', attribute_0, attribute_1, attribute_2, attribute_3)
-    insert all events into OutputStream;
-
-This query uses a Bayesian regression model named `model1` to predict
-the label of the feature vector represented by `attribute_0`,
-`attribute_1`, `attribute_2`, and `attribute_3`. The predicted value is
-emitted to the `OutputStream` streamalong with the prediction confidence
-(std of predictive distribution) and the feature vector. As a result,
-the OutputStream stream is defined as follows: (attribute_0 double,
-attribute_1 double, attribute_2 double, attribute_3 double,
-prediction double, confidence double).
+```sql
+CREATE STREAM StreamA (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double);
+CREATE STREAM PredictionStream (attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double);
+
+CREATE SINK STREAM OutputStream (prediction double, confidence double, attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double);
+
+@info(name = 'predictAttributes')
+INSERT INTO OutputStream
+SELECT prediction, confidence, attribute_0, attribute_1, attribute_2, attribute_3
+FROM PredictionStream#streamingml:bayesianRegression('model1', attribute_0, attribute_1, attribute_2, attribute_3);
+```
+
+This query uses a Bayesian regression model named `model1` to predict the label of the feature vector represented by `attribute_0`, `attribute_1`, `attribute_2`, and `attribute_3`. The `StreamA` contains the input features, and the `PredictionStream` is created with the same features.
+
+A sink stream, `OutputStream`, is defined to store the predicted values, prediction confidence (std of predictive distribution), and the original features from the `PredictionStream`.
+
+The `predictAttributes` query processes events from the `PredictionStream`, using the Bayesian regression model 'model1' to make predictions based on the input features. The predicted values, along with their confidences and original features, are inserted into the `OutputStream` sink stream. As a result, the OutputStream stream is defined as follows: (prediction double, confidence double, attribute_0 double, attribute_1 double, attribute_2 double, attribute_3 double).
```

Comment:
We cannot use Example 1 directly we need to add somewhere that before using any bayesianRegression model we need to initialise it first using updateBayesianRegression... and that's why example2 is good.